### PR TITLE
Last use cases!

### PR DESCRIPTION
**Pull request by catlook**

### CODE REVIEW

**Comment by catlook** [comment]
Closes issue #22

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-e18edf021d/whitepapers/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3d3d61e542/whitepapers/)

File: content/use-cases/a-high-street-alternative-in-the-rackspace-hybrid-cloud.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: A High Street Alternative in the Rackspace Hybrid Cloud
 type: case_study
```
Comment (nimbinatus) [inline content/use-cases/a-high-street-alternative-in-the-rackspace-hybrid-cloud.md]: `type` should be `whitepaper`

File: content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+---
+permalink:usourceit-reduces-clients-it-costs-by-50-percent/
```
Comment (nimbinatus) [inline content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md]: space needed between `:` and `usourceit...`

File: content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
+---
+permalink: welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo/
```
Comment (nimbinatus) [inline content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md]: Should the `m` in `kingdom` be there at the end?

Comment (catlook) [inline content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md]: :shrug: that's how it generated.  How important is this to you?

File: content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md (hunk 2 above)
Comment (catlook) [inline content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md]: just wondering why?  Looks like it built and displays.  I'll do it, but is it necessary?  or just nice?

File: content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md (hunk 3 above)
Comment (nimbinatus) [inline content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md]: Not all that important; I was just wondering, really.

File: content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md (hunk 2 above)
Comment (nimbinatus) [inline content/use-cases/usourceit-reduces-clients-it-costs-by-50-percent.md]: It's not showing up in the list on the landing page in the preview. The space is needed so the template for the landing page can parse the permalink and put it up there.

File: content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md (hunk 3 above)
Comment (catlook) [inline content/use-cases/welcome-to-yorkshire-creates-innovative-marketing-and-pr-campaigns-in-the-united-kingdo.md]: heh... me too...but that's what the file is named, so it caries over!

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5fac5476d0/whitepapers/)

**Comment by catlook** [comment]
thanks for explaining why the space was needed after teh colon!

**Comment by catlook** [comment]
good to go?

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-cc0bc7d4e3/whitepapers/)

**Comment by nimbinatus** [comment]
Strider got hung up; had to go redeliver it. It's good now that the preview appeared after your last commit.

**Pull request merged**